fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -3605,9 +3605,9 @@ show_param ::=
         RESULT = new ShowTableStatsStmt(tbl, partitionNames);
     :}
     /* show column stats */
-    | KW_COLUMN KW_STATS table_name:tbl opt_partition_names:partitionNames
+    | KW_COLUMN KW_STATS table_name:tbl
     {:
-        RESULT = new ShowColumnStatsStmt(tbl, partitionNames);
+        RESULT = new ShowColumnStatsStmt(tbl);
```

Comment:
why remove partition level column stats?

Reply:
Since I thought column statistics is all user need to care about, I'll recover the support for partition level stats soon